<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Остров сокровищ</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.jpg</normaloff>icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>440</width>
      <height>180</height>
     </rect>
    </property>
    <property name="title">
     <string>Сервер</string>
    </property>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Персонаж</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Очерёдность клиента</string>
     </property>
    </widget>
    <widget class="QDial" name="waitDial">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>30</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchesVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="waitLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ожидание подлючения клиентов!</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="clientQueue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="heroNumber">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Отправить сигнал</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
